README.md

```diff
@@ -106,7 +106,45 @@ Why Jekyll? Read [Andrej Karpathy's blog post](https://karpathy.github.io/2014/0
 
 ### Installation
 
-#### Local setup
+#### Local setup using Docker (Recommended)
+
+You need to do these step to get `ai-folio` up and running in your local machine:
+
+- Install [docker](https://docs.docker.com/get-docker/)
+- Clone your repository
+
+```bash
+      git clone git@github.com:<your-username>/<your-repo-name>.git
+      cd <your-repo-name>
+```
+
+Then you can use Dockerhub prebuilt image to run your website. Note that for the first time it would download an image of size 300MB or so. 
+
+```bash
+      bin/dockerhub_run.sh
+```
+
+##### More Advanced Users
+You can also build your own docker image. 
+
+So first, download necessary modules and install them into a docker image called `mywebsite:Dockerfile` (this command will build an image which is used to run your website afterwards. Note that you only need to do this step once. After you have the image, you no longer need to do this anymore):
+  
+
+```bash
+    bin/docker_build_image.sh  
+```
+
+Run the website!
+
+```bash
+    bin/docker_run.sh
+```
+
+> In order to change port number, you can change `docker_run.sh` file.
+
+> If you want to update jekyll or install new ruby packages or ... , all you have to do is to build the image again using `docker_build_image.sh`! It will download ruby and jekyll and installall ruby packages again.
```

Comment:
let's hide the advanced users section using `<details>`
```suggestion
<details><summary>(click to expand) <strong>Build your own docker image (more advanced):</strong></summary>

First, download the necessary modules and install them into a docker image called `mywebsite:Dockerfile` (this command will build an image which is used to run your website afterwards. Note that you only need to do this step once. After you have the image, you no longer need to do this anymore):
  

\```bash
    bin/docker_build_image.sh  
\```

Run the website!

\```bash
    bin/docker_run.sh
\```

> To change port number, you can edit `docker_run.sh` file.

> If you want to update jekyll, install new ruby packages, etc., all you have to do is build the image again using `docker_build_image.sh`! It will download ruby and jekyll and install all ruby packages again from scratch.

</details>
```